include/rmm/cuda_stream_view.hpp

```diff
@@ -58,16 +70,22 @@ class cuda_stream_view {
 
   /**
    * @brief Implicit conversion to cudaStream_t.
+   *
+   * @return cudaStream_t The wrapped stream.
    */
   constexpr operator cudaStream_t() const noexcept { return value(); }
 
   /**
    * @brief Return true if the wrapped stream is the CUDA per-thread default stream.
+   *
+   * @return true if the wrapped stream is the CUDA per-thread default stream.
```

Comment:
Can we use the brief as the return rather than duplicating?

```suggestion
   * @return true if the referenced stream is the CUDA per-thread default stream.
```